css-layout-classes/index.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <div class="container">
+      <div class="row">
+        <h1>Choose your character!</h1>
+      </div>
+      <div class="row">
+        <img class="character column-third" src="images/link.png" alt="Link" />
```

Comment:
This `img` should be inside a `div` that has the class `column-third`. The `img` shouldn't have the column class